google_application_credentials.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Is there a way we can avoid checking a file in that has what looks like a private key in it. Like, for instance, can we include instructions on how to generate this file? I understand that it is not meant to be secure but I'm just worried we may get some heat for how this file looks?

Reply:
ONS reached out to me this morning soon after the commit as I suspected they would. They were satisfied with my explanation. If you have further doubts then read the discussion linked to in the readme file.

It is annoying that Firestore insists on default application credentials like this, other NoSQL database do not, but there you go.

Reply:
Ok, fair enough.